### PR TITLE
[merged] doc: Added flannel config example.

### LINKED ISSUE #87: Docs should say something about flannel config
I finally solved my CPU spikes and failed bootstraps.

The journal on the host machine was filled with:
```
Apr 05 01:28:10 atomic-00 flanneld[826]: E0405 01:28:10.644974 00826 network.go:53]
Failed to retrieve network config: 100: Key not found (/atomic01/network/config) [8420]
```

Following https://coreos.com/flannel/docs/latest/flannel-config.html, I did:
```
$ etcdctl set /atomic01/network/config '{ "Network": "10.1.0.0/16" }'
```
and that solved it.

The **Getting Started** docs should say *something* about this, especially since the `/atomic01/network` etcd path is hard-coded.

### PR DESCRIPTION
**Pull request by ashcrow**
This change closes #87.

### CODE REVIEW

File: doc/gettingstarted.rst [hunk 1]
```diff
@@ -26,6 +26,13 @@ the unittests run. From the repo root...
 .. include:: examples/run_unittest_example.rst
 
 
+Setup Overlay Network Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Flannel requires a configuration inside of etcd.
+
+.. include:: examples/flannel_overlay_network_example.rst
+
+
```
Comment (mbarnes) [inline doc/gettingstarted.rst]: Should we also add:

See https://coreos.com/flannel/docs/latest/flannel-config.html for more details about Flannel configuration.

**Comment by ashcrow** [comment]
@mbarnes Good idea. I'll also link the http://www.projectatomic.io/docs/gettingstarted/ in a fixup.

File: doc/examples/flannel_overlay_network_example.rst [hunk 2]
```diff
@@ -1,3 +1,7 @@
+.. note::
+
+   For more information as to why this is necissary see the `flannel documentation <https://coreos.com/flannel/docs/latest/flannel-config.html>`_ or the `Project Atomic Getting Started Guide <http://www.projectatomic.io/docs/gettingstarted/>`_
```
Comment (mbarnes) [inline doc/examples/flannel_overlay_network_example.rst]: Wording looks good, except "necissary"  :smile:

**Comment by ashcrow** [comment]
:arrow_up:

**Comment by mbarnes** [comment]
:+1:

@cgwalters-bot r+

**Pull request closed without merging**